packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -195,7 +195,8 @@ const baseConfig = (target) => {
                         DEBUG,
                         NODE_ENV: `'${process.env.NODE_ENV}'`,
                         WEBPACK_TARGET: `'${target}'`,
-                        ['global.GENTLY']: false
+                        ['global.GENTLY']: false,
+                        SFDC_EXT_MARKETING_CONSENT_ENABLED: true
```

Comment:
This is set via user-selection when the generator is executed. Not as part of webpack config

Reply:
Right, we do need this as a global variable, otherwise compiler will fail due to it's undefined. 
I wonder if we can just do it as a global declaration somewhere (like global.d.ts) instead of as a webpack plugin. I will take a look. But the code is OK as is for now.